examples/deepcopy-gen/output_tests/maps/zz_generated.go

```diff
@@ -104,13 +104,152 @@ func (in *Ttest) DeepCopyInto(out *Ttest) {
 		in, out := &in.StringPtr, &out.StringPtr
 		*out = make(map[string]*string, len(*in))
 		for key, val := range *in {
+			var outVal *string
 			if val == nil {
 				(*out)[key] = nil
 			} else {
-				x := new(string)
-				(*out)[key] = x
-				*x = *val
+				in, out := &val, &outVal
+				if *in == nil {
```

Comment:
This is redundant with L107/108 above

Reply:
fixed. Pointers had a redundant nil check.